tests/components/evohome/conftest.py

```diff
@@ -23,6 +22,9 @@
 
 from tests.common import load_json_array_fixture, load_json_object_fixture
 
+if TYPE_CHECKING:
+    from evohomeasync2.broker import Broker
+
```

Comment:
What is the benefit of this?
We usually avoid TYPE_CHECKING checks unless required for circular imports.

Reply:
It's definitely not circular.

I do it as a habit, so I know what imports are only for typing.

It's a ruff linter rule, `TCH002`.

Reply:
I must be misunderstanding something... If I look at **pyproject.toml**:
```
[tool.ruff.lint]
select = [
...
    "TCH", # flake8-type-checking
```

Reply:
Some sub-rules purposefully ignored in
https://github.com/home-assistant/core/blob/b996bd3e65d43cb9cf54a956228554cd6a2c0c08/pyproject.toml#L809-L812